examples/msi-renamed-to-authorization/index.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright 2016-2017, Pulumi Corporation.  All rights reserved.
```

Comment:
Nit: Should this file be in `step1` folder?

Reply:
I am copying this from what @CyrusNajmabadi setup in pulumi-aws. Where we assume this is step1 and then apply a real step2 on top